crates/node/src/shell/finalize_block.rs

```diff
@@ -1710,6 +1711,27 @@ mod test_finalize_block {
         });
     }
 
+    /// Test the correct transition to a new masp epoch
+    #[test]
+    fn test_masp_epoch_progression() {
+        let (mut shell, _broadcaster, _, _eth_control) = setup();
+
+        let masp_epoch_multiplier =
+            namada::ledger::parameters::read_masp_epoch_multiplier_parameter(
+                &shell.state,
+            )
+            .unwrap();
+
+        assert_eq!(shell.state.get_block_epoch().unwrap(), Epoch::default());
```

Comment:
the default epoch is actually 1 I think, no? or is it 0 :confused:

Reply:
We derive the `Default` trait and `Epoch` it's just a wrapper around a `u64`, os it should be 0